builtin/mainmenu/tab_local.lua

```diff
@@ -269,6 +270,14 @@ local function main_button_handler(this, fields, name, tabdata)
 	end
 
 	if fields["play"] ~= nil or world_doubleclick or fields["key_enter"] then
+		if is_in_dlg_create_world then
+			is_in_dlg_create_world = false
+
+			if fields["play"] == nil and not this.is_dlg_create_world_canceled then
```

Comment:
If you create a world without pressing enter then try to play it by pressing enter on it, nothing happens. Another press of enter is necessary.

Reply:
Thank you for the catch. It should have been fixed now.